packages/eslint-plugin/tests/rules/component-selector/cases.ts

```diff
@@ -302,9 +312,9 @@ export const invalid = [
         })
         class Test {}
       `,
-    messageId: messageIdStyleFailure,
+    messageId: messageIdStyleAndPrefixFailure,
```

Comment:
Note: the reported error for this invalid case has changed to the new error message. Although it's not incorrect, the former is more concise for this particular case. I think the only way to retain the original message is to move the style logic inside the prefix validator to its own validator (which I did attempt in a previous commit). However, this will result in a more complex conditional in the rule when determining which message to report.

LMK your thoughts!

Reply:
This seems acceptable to me as a non breaking change while addressing the original issue.